graph_builder.ts

```diff
@@ -52,7 +52,7 @@ export function buildGraph(
   >,
   sourceToBundles: { [sourceName: string]: Set<string> },
   logger: Logger
-) {
+): SourceMapGraph {
```

Comment:
the SourceMapGraph type can be inferred by the typescript compiler no need to specify